client/.storybook/addons.js

```diff
@@ -1,2 +1,4 @@
 import '@storybook/addon-actions/register';
 import '@storybook/addon-links/register';
+import '@storybook/addon-viewport/register';
+import '@storybook/addon-actions/register';
```

Comment:
You already hast one ```addon-actions/register```

Reply:
Resolved in 25555fb615c34752375df32328752db628c4b4b1